mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -880,11 +880,13 @@ class LDA @Since("1.6.0") (
   }
 }
 
-
-private[clustering] object LDA extends DefaultParamsReadable[LDA] {
+@Since("2.0.0")
+object LDA extends DefaultParamsReadable[LDA] {
```

Comment:
yeah, seems we have generally had `load` be since the version that model loading was actually added for that model? e.g. I see that is the case for tree models etc